### PR TITLE
Initial version of README.md

### PR DESCRIPTION
**Pull request by phako**

### CODE REVIEW

**Comment by nshehz** [comment]
Looks fine. I wish rather than TMCB I would have used more generic name for a device.
snipptes -> snippets
chosed  -> chosen.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+The configuration file for the control system depends on the chosed control
+system adapter. If no file is provided, it will default to expose everything to
+the control system.
+
+#### DOOCS
+
+The file for DOOCS has to be named
+`generic_chimeratk_server-DoocsVariableConfig.xml`. It might also be advisable,
+though not necessary, to provide a DOOCS server configuration file containing
```
Comment (killenb) [inline README.md:150]: It is necessary to have a doocs server config to specify the RPC number. The DoocsVariableConfig.xml however is optional.

**Review by killenb — COMMENTED** [review]
Very nice writeup. Appoved after fixing the section with the doocs server config.

File: README.md (hunk 1 above)
Comment (phako) [reply README.md]: Reworded the DOOCS section

File: README.md [hunk 2]
```diff
@@ -139,16 +139,18 @@ And then we just close off the file.
...
 system adapter. If no file is provided, it will default to expose everything to
 the control system.
 
 #### DOOCS
 
-The file for DOOCS has to be named
-`generic_chimeratk_server-DoocsVariableConfig.xml`. It might also be advisable,
-though not necessary, to provide a DOOCS server configuration file containing
-permissions and such. For details, refer to the [documentation of the DOOCS
+DOOCS requires a basic server configuration file named `doocs-generic-chimeratk-server01.conf` to set `SVR.RPC_NUMBER`
```
Comment (killenb) [inline README.md:148]: The question is how much information we want to give about configuring a DOOCS server at all. A minimal version to get a server running at all looks like this:
```
eq_conf: 

eq_fct_name: 	"LOCAHOST._SVR"
eq_fct_type: 	1
{
SVR.RPC_NUMBER:  	610498009
}
```
However, it is not really usable in production context. The minimum you want is to set SVR.RATE, SVR.STORE.RATE, and SVR.STORE.AUTO. These are needed to have the input data persisted (I am a bit unsure which ones are really needed, but I think if SVR.RATE is not set, update is never called and the config file is never written.

We should clean up the config file to the minimum reasonable. It looks something like this (taken from piezo server):
```
eq_conf: 

oper_uid:   -1
oper_gid:   ${OPERATOR_GID}
xpert_uid:  0
xpert_gid:  0

eq_fct_name:    "${SVR_LOCATION}"
eq_fct_type:    1
{
SVR.RATE:          1  0  0  0
SVR.PROGRAMMER:    "M. Killenberg"
SVR.STORE.RATE:    60
SVR.STORE.AUTO:    10
SVR.HOST_NAME:     ${HOSTNAME}
SVR.RPC_NUMBER:    ${rpcNumber}
SVR.WDADDR:        "${WATCHDOG_ADDRESS}"
SVR.FACILITY:      "${FACILITY}"
SVR.DEVICE:        "${DEVICE}"
}
```

File: README.md [hunk 3]
```diff
@@ -139,16 +139,18 @@ And then we just close off the file.
...
 #### DOOCS
 
-The file for DOOCS has to be named
-`generic_chimeratk_server-DoocsVariableConfig.xml`. It might also be advisable,
-though not necessary, to provide a DOOCS server configuration file containing
-permissions and such. For details, refer to the [documentation of the DOOCS
+DOOCS requires a basic server configuration file named `doocs-generic-chimeratk-server01.conf` to set `SVR.RPC_NUMBER` 
+
+Additional configuration can be done through the DOOCS variable mapping file. It has to be named
+`generic_chimeratk_server-DoocsVariableConfig.xml`. If omitted, all variables are exported to DOOCS.
```
Comment (killenb) [inline README.md:151]:
```suggestion
`generic_chimeratk_server-DoocsVariableConfig.xml`. If omitted, all variables are exported to DOOCS, with the highest hierarchy level becoming the location name, and the other levels being converted into a dot-separated property name (i.e. /Module2/SubModule3/property7 becomes DOOCS property //Module2/SubModule3.property7)
```

File: README.md (hunk 2 above)
Comment (killenb) [reply README.md:148]: You only specify the server location here. The other locations are added at run time by the generic server.

**Review by killenb — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (phako) [reply README.md]: I have removed the whole doocs section and just referred to the DOOCS CSA docs

File: README.md (hunk 3 above)
Comment (phako) [reply README.md]: This should go into the DOOCS CSA, where this information is missing as well

**Review by killenb — APPROVED** [review]
Did you force-push the branch? I could not compare to the state of my last review because the revision could not be found?

Looks good to me.

**Pull request merged**